dbus/methods.go

```diff
@@ -221,6 +221,36 @@ func (c *Conn) GetUnitTypeProperty(unit string, unitType string, propertyName st
 	return c.getProperty(unit, "org.freedesktop.systemd1."+unitType, propertyName)
 }
 
+// ListUnitsFiltered returns an array with units.
+// It takes a list of units' statuses and names to filter.
+// Note that units may be known by multiple names at the same time,
+// and hence there might be more unit names loaded than actual units behind them.
+func (c *Conn) ListUnitsFiltered(states []string, patterns []string) ([]UnitStatus, error) {
+	result := make([][]interface{}, 0)
+	err := c.sysobj.Call("org.freedesktop.systemd1.Manager.ListUnitsFiltered", 0, states, patterns).Store(&result)
```

Comment:
is this documented somewhere? seems they've stopped updating the API page :-( https://www.freedesktop.org/wiki/Software/systemd/dbus/